theblues/charmstore.py

```diff
@@ -361,6 +401,10 @@ def _common_query_parameters(self, doc_type, includes, owner,
         return queries
 
     def fetch_related(self, ids):
+        """Fetch related entity information.
+
+        @param ids The entity ids to fetch related information for.
```

Comment:
Also, this params really does not explain what type is expected here. I thought it was just an iterator of strings, but looking at the code it does not seem true.